### PR TITLE
Add changelog 7.0.0

### PR DESCRIPTION
**Pull request by will-moore**
Changelog 7.0.0

See https://github.com/will-moore/figure/blob/changelog_7.0.0/CHANGELOG.md

cc @jburel

### CODE REVIEW

**Comment by will-moore** [comment]
@jburel As mentioned in the meeting today - just waiting on your review of this to move forward with figure release

**Comment by will-moore** [comment]
@jburel - ping for review of this changelog, thx

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+7.0.0 (May 2024)
+----------------
+
+- Update JavaScript build tools and all JavaScript libraries ([#477](https://github.com/ome/omero-figure/pull/477))
+    - This includes an update to the Bootstrap UI framework and Icons library.
+	- Please see the PR for more details of the changes
+	- Follow-up bug-fix: [#561](https://github.com/ome/omero-figure/pull/561)
+- Fix the layering of ROI Shapes so that smaller shapes are always on top and clickable.
+    - NB: This was fixed in the separate Shape-Editor repo at [#12](https://github.com/ome/shape-editor/pull/12) (thanks to Rémy Dornier) and this library now been moved into the omero-figure repo in #477 above.
```
Comment (jburel) [inline CHANGELOG.md:9]:
```suggestion
    - NB: This was fixed in the separate Shape-Editor repo at [#12](https://github.com/ome/shape-editor/pull/12) (thanks to Rémy Dornier) and this library has now been moved into the omero-figure repository in #477 above.
```

**Comment by jburel** [comment]
You may want to change the year in the README, it is still 2022

**Comment by will-moore** [comment]
Thanks - fixed.

**Comment by jburel** [comment]
@will-moore can you add https://github.com/ome/omero-figure/pull/563? then release

**Pull request merged**